tests/examples/CMakeLists.txt

```diff
@@ -256,26 +258,26 @@ function(run_test test_path namespace escaped_content)
 
     file(RELATIVE_PATH rel_template "${TOP_SOURCE_DIR}" "${template}")
     file(RELATIVE_PATH rel_test_path "${TOP_SOURCE_DIR}" "${test_path}")
+
+    # Add target to run this as test (via check-examples, not ignoring errors).
+    set(TARGET_NAME "run-${rel_test_path}")
+    STRING(REPLACE "/" "-" TARGET_NAME ${TARGET_NAME})
+    set(EXAMPLE_DOC_TEST_TARGETS ${EXAMPLE_DOC_TEST_TARGETS} ${TARGET_NAME}
+        PARENT_SCOPE)
+    add_custom_target(${TARGET_NAME}
+        COMMAND "${TOP_SOURCE_DIR}/tests/examples/runner.sh" "${expected_output_path}" ${LUA_COV_RUNNER} ${template} ${test_path} ${IMAGE_PATH}
+        VERBATIM)
```

Comment:
This generates a lot of internal targets (bloating the generated Makefile).
Is there a better way?
Maybe this should get moved into a wrapper script so that only a single command/target is needed?
On the other hand it can be useful to run a specific test only (e.g. for coverage debugging), but that could be left commented out then also.

Reply:
> This generates a lot of internal targets (bloating the generated Makefile).

Why is that bad?

The current version allows to run these targets in parallel.

Reply:
It is not really bad, but each target adds like 10-20 lines there etc, and I got the impression that it would also slow down `make clean`, but have not investigated.
I think it is much better than the current version anyway.